src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.html

```diff
@@ -8,7 +8,7 @@
       <li class="list-group-item">
         <cd-card-row [data]="rgwDaemonCount"
                      link="/rgw/daemon"
-                     title="Daemon"
+                     title="Service"
```

Comment:
We decided to call them Gateway not long ago right?
```suggestion
                     title="Gateway"
```

Reply:
Ack, missed it!